com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/validators/MtaArchiveValidatorTest.java

```diff
@@ -35,7 +36,7 @@
 
     private static final long MAX_MTA_DESCRIPTOR_SIZE = 1024 * 1024l; // 1 MB
 
-    private static final String XS2_PLATFORM_NAME = "initial initial";
+    private static final String XS2_PLATFORM_NAME = "XS2-INITIAL";

```

Comment:
Why is this changed?

Reply:
Code must cover both cases but just in case i added tests for both scenarios.